src/components/routerView.vue

```diff
@@ -31,16 +30,15 @@
   provide(depthInjectionKey, depth + 1)
 
   const component = computed(() => {
-    const routeComponent = router.route.matches[depth].component
+    const routeComponent = router.route.matches[depth]?.component
 
     if (routeComponent) {
       if (typeof routeComponent === 'function') {
         return defineAsyncComponent(routeComponent as AsyncComponentLoader)
       }
-
       return routeComponent
     }
 
-    return routerView
+    return 'span'
```

Comment:
Why a span?

Reply:
what should it be? alternatively I could have component return `undefined` and add a `v-if` in the template?

Reply:
Oh this is the noop? I think you can just turn null. Then it won't render anything